src/metabase/pulse/render/body.clj

```diff
@@ -548,6 +585,8 @@
                                 :width   :100%})
            :src   (:image-src image-bundle)}]]})
 
+(def ^:dynamic *axis-group-threshold* 0.5)
```

Comment:
I thought you said `0.33`? Or is this a different number?

Reply:
Also, does it need to be dynamic?

Reply:
Doesn't need to be dynamic, no, now that I think about it. This was also just a mistake taht I checked in 0.5. I had changed it while playing around in the REPL, and it'll be changed back